demo/src/screens/componentScreens/BadgesScreen.js

```diff
@@ -154,7 +161,7 @@ export default class BadgesScreen extends Component {
               border radius 2
             </Text>
           </View>
-        </View>
+        </View> */}
```

Comment:
You can just delete the unneeded examples, no need to comment them out 👍

Reply:
Yes yes, that was a temp comment and I forgot to remove it.
My bad :)